src/utils/pools.spec.ts

```diff
@@ -5,29 +5,40 @@ import { getProvider } from '../crypto/providers';
 import { ChainId } from '../crypto/networks';
 import { getPoolReserves } from './pools';
 import { getVault } from '../constants/vaults';
-import {BigNumber} from '@ethersproject/bignumber';
 import { VaultName } from '../enums/vaultName';
+import { MainnetPoolNumbers } from '../enums/poolNumber';
+
+const testChainId = ChainId.Mainnet
+const normalAddress = getVault(VaultName.USDC_ICHI, ChainId.Mainnet).address;
+const normalPoolID: MainnetPoolNumbers = MainnetPoolNumbers.USDC_VAULT;
 
-const normalAddress = getVault(VaultName.FUSE, ChainId.Mainnet).address;
 const exceptionVaultAddress = getVault(VaultName.ICHI, ChainId.Mainnet).address;
+const exceptionPoolID: MainnetPoolNumbers = MainnetPoolNumbers.ONE_UNI_VAULT_LP;
+
+console.log("Infura ID was " + process.env.INFURA_ID);
 
 describe('utils/pools', () => {
-  describe('Check Pool Reserves', async () => {
-    const provider = await getProvider(ChainId.Mainnet);
-    if (!provider) {
-      throw new Error(`Could not get provider`);
-    }
+  describe('Check Pool Reserves', () => {
     it('Should use totalAmounts for normal case', async () => {
-      let ichiVault = getIchiVaultContract(normalAddress, provider);
-      let {_reserve0, _reserve1} = await getPoolReserves(ichiVault, ChainId.Mainnet);
+      const provider = await getProvider(ChainId.Mainnet) 
+      if (!provider) {
+        console.log("Could not connect with provider");
```

Comment:
throw e here, if the provider couldn't be connected to then all subsequent falls will fail